changes/1895.bugfix.rst

```diff
@@ -0,0 +1 @@
+Modified to properly implement `current_window` on windows platform.
```

Comment:
This should be a misc, not bug fix; it's an update to a bugfix that is already listed in the in-progress release.
```suggestion
The implementation of ``current_window`` on Winforms was corrected.
```